vector/src/main/res/layout/item_bottom_sheet_action.xml

```diff
@@ -82,4 +82,27 @@
         tools:ignore="MissingPrefix"
         tools:visibility="visible" />
 
+
+    <TextView
+        android:id="@+id/actionBetaTextView"
+        style="@style/Widget.Vector.TextView.Caption"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:background="@drawable/notification_badge"
+        android:backgroundTint="@color/palette_azure"
+        android:gravity="center"
+        android:paddingStart="10dp"
+        android:paddingTop="3dp"
+        android:paddingEnd="10dp"
+        android:layout_marginEnd="1dp"
+        android:paddingBottom="3dp"
+        android:visibility="gone"
+        tools:visibility="visible"
```

Comment:
please rearrange attributes

Reply:
Yes, also maybe we can force `ktlintformat` to do that if possible

Reply:
ktlint only works on Kotlin files AFAIK, not XML resources.
But if you found a solution for this, it could be very nice.